dropshots/src/main/java/com/dropbox/dropshots/Dropshots.kt

```diff
@@ -205,19 +296,25 @@ public class Dropshots internal constructor(
     filePath: String?,
     screenshot: Bitmap,
     referenceImage: Bitmap,
-    mask: Mask?
+    mask: Mask?,
   ): String {
     val screenshotFolder = File(rootScreenshotDirectory, "diff".appendPath(filePath))
     val diffImage = generateDiffImage(referenceImage, screenshot, mask)
-    return writeImage(screenshotFolder, name, diffImage)
+    return writeImage(screenshotFolder, name, diffImage, "_diff")
   }
 
-  private fun writeImage(dir: File, name: String, image: Bitmap): String {
+  private fun writeImage(dir: File, name: String, image: Bitmap, nameSuffix: String = ""): String {
```

Comment:
Instead of modifying the name of the diff files that are written, which people might have automated systems to handle, let's continue using the existing directories to differentiate. This function already writes diffs to `File(rootScreenshotDirectory, "diff".appendPath(filePath))` and reference images to `File(rootScreenshotDirectory, "reference".appendPath(filePath))` and the plugin already pulls from those directories, so I don't see a compelling case to changing the diff filename, which will impact any integrations people already have. 

Instead, we can just update the plugin to pull the reference images to a `reference` subdirectory in the build root, let the diff writing all happen the same way it does now, and then this will be fully backwards compatible.